Singularity/Singularity/Screen/ELogEventType.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Runtime.Serialization;
+
+namespace Singularity.Screen
+{
+    public enum ELogEventType
+    {
+        [EnumMember(Value = "Unit attacked")]
+        UnitAttacked,
+
+        [EnumMember(Value = "Platform attacked")]
+        PlatformAttacked,
+
+        [EnumMember(Value = "Platform built")]
+        PlatformBuilt,
+
+        [EnumMember(Value = "Unit built")]
```

Comment:
Is that triggered? If not, does someone who knows where to trigger it know how to trigger it / an Issue requesting this feature exist?

Reply:
Same with Platform Destroy, Low Energy, Info (for the Story maybe?)

Reply:
Not yet